pkg/cvo/cvo_test.go

```diff
@@ -3272,169 +3272,64 @@ func fakeClientsetWithUpdates(obj *configv1.ClusterVersion) *fake.Clientset {
-								},
-							},
-						},
-					},
-				},
-			},
-			wantErr: true,
+			name:     "requires verifiers",
+			fileName: "requires-verifiers.yaml",
+			update:   &payload.Update{},
+			wantErr:  true,
 		},
 		{
-			name: "loads valid configuration",
-			update: &payload.Update{
-				Manifests: []lib.Manifest{
-					{
-						GVK: schema.GroupVersionKind{Version: "v1", Kind: "ConfigMap"},
-						Obj: &unstructured.Unstructured{
-							Object: map[string]interface{}{
-								"metadata": map[string]interface{}{
-									"name":      "release-verification",
-									"namespace": "openshift-config-managed",
-									"annotations": map[string]interface{}{
-										"release.openshift.io/verification-config-map": "",
-									},
-								},
-								"data": map[string]interface{}{
-									"verifier-public-key-redhat": string(redhatData),
-									"store-local":                "file://../verify/testdata/signatures",
-								},
-							},
-						},
-					},
-				},
-			},
+			name:          "loads valid configuration",
+			fileName:      "loads-valid.yaml",
+			update:        &payload.Update{},
 			want:          true,
 			wantVerifiers: 1,
 		},
 		{
-			name: "only the first valid configuration is used",
-			update: &payload.Update{
-				Manifests: []lib.Manifest{
-					{
-						GVK: schema.GroupVersionKind{Version: "v1", Kind: "ConfigMap"},
-						Obj: &unstructured.Unstructured{
-							Object: map[string]interface{}{
-								"metadata": map[string]interface{}{
-									"name":      "release-verification",
-									"namespace": "openshift-config-managed",
-									"annotations": map[string]interface{}{
-										"release.openshift.io/verification-config-map": "",
-									},
-								},
-								"data": map[string]interface{}{
-									"verifier-public-key-redhat": string(redhatData),
-									"store-local":                "\nfile://../verify/testdata/signatures\n",
-								},
-							},
-						},
-					},
-					{
-						GVK: schema.GroupVersionKind{Version: "v1", Kind: "ConfigMap"},
-						Obj: &unstructured.Unstructured{
-							Object: map[string]interface{}{
-								"metadata": map[string]interface{}{
-									"name":      "release-verificatio-2n",
-									"namespace": "openshift-config-managed",
-									"annotations": map[string]interface{}{
-										"release.openshift.io/verification-config-map": "",
-									},
-								},
-								"data": map[string]interface{}{
-									"verifier-public-key-redhat":   string(redhatData),
-									"verifier-public-key-redhat-2": string(redhatData),
-									"store-local":                  "file://../verify/testdata/signatures",
-								},
-							},
-						},
-					},
-				},
-			},
+			name:          "only the first valid configuration is used",
+			fileName:      "only-first-used.yaml",
+			update:        &payload.Update{},
 			want:          true,
 			wantVerifiers: 1,
 		},
 	}
 	for _, tt := range tests {
+		if tt.fileName != "no-op.yaml" {
```

Comment:
possibly `s/no-op.yaml//`, because an empty string is a clearer marker of "don't load any filenames" to me than `no-op.yaml`.  But obviously works either way, so feel free to leave it as it stands if you feel `no-op.yaml` is more intuitive.

Reply:
Changed.